docs/src/reference/nonlinear.md

```diff
@@ -31,9 +31,13 @@ features_available
 eval_objective
 eval_constraint
 eval_objective_gradient
+hessian_objective_structure
+eval_hessian_objective
```

Comment:
So we have `eval_objective_gradient`, but `eval_hessian_objective`. 

Perhaps it should be `eval_objective_hessian`?